workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/Partials/Improbable.Vector3d

```diff
@@ -0,0 +1,97 @@
+/// <summary>
```

Comment:
If we are implementing all of these we should also do IEquatable et al.

Reply:
Agreed - this should be a wider pass over all generated types